Vimeo-Native/VimeoInterop/VimeoInterop.vcxproj

```diff
@@ -208,7 +208,7 @@ popd</Command>
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
     <Reference Include="VimeoAPI">
-      <HintPath>..\packages\VimeoAPI.1.0.1.0\lib\net40\VimeoAPI.dll</HintPath>
+      <HintPath>..\packages\VimeoAPI.2.0.0.0\lib\net46\VimeoAPI.dll</HintPath>
```

Comment:
Does this dll get installed to a "_noversion" folder? If so, can you update this reference to use that path instead?

That avoids having to update this path in the future when we update the VimeoAPI package.